sys/tiny_strerror/tiny_strerror.c

```diff
@@ -117,6 +118,12 @@ const char *tiny_strerror(int errnum)
         errnum = -errnum;
     }
 
+    if (IS_USED(MODULE_TINY_STRERROR_MINIMAL)) {
+        static char buf[4];
```

Comment:
This is not thread-safe. I don't think this is much of an issue, as in few cases more than one thread cares about user-friendly error messages. And if so they would have to interrupt each other during error reporting. And even then likely they report errors via stdio, which gets garbled anyway when two threads concurrently write to it.

Still, I think this is something we need to point out clearly with a big fat `@warning`.